core/style.css

```diff
@@ -836,6 +836,23 @@ h3.title {
   font-size: 16px;
 }
 
+/* history details */
+#historydetails {
+  text-align: center;
+  color: #ffce00;
+}
+
+#historydetails .missing {
+}
+
+#historydetails span {
+  color: #ff4a4a;
```

Comment:
I'm not sure about exact values.
We should use the same as in other iitc styles, and I am unable to find `#ff4a4a`/`#03fe03`.